src/utils/bitstream.rs

```diff
@@ -1,72 +1,75 @@
 //! This module provides a bit reader
 
-use std::old_io;
-use std::old_io::{IoResult};
+use std::io::{
+    self,
```

Comment:
@nwin It was added months ago.
See https://github.com/rust-lang/rust/pull/20365